cli/operator/node.go

```diff
@@ -140,57 +142,73 @@ var StartNodeCmd = &cobra.Command{
 
 		var operatorPrivKey keys.OperatorPrivateKey
 		var operatorPrivKeyText string
-		if cfg.KeyStore.PrivateKeyFile != "" {
-			// nolint: gosec
-			encryptedJSON, err := os.ReadFile(cfg.KeyStore.PrivateKeyFile)
+		var operatorPubKey keys.OperatorPublicKey
+		var ssvSignerClient *ssvsignerclient.SSVSignerClient
```

Comment:
we need to add a check here that only one way of passing the key is used. otherwise we crash, in order to minimize user error.
Text/Keystore/Signer

Reply:
done